catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/RemoveCommand.java

```diff
@@ -118,7 +118,8 @@ private Object executeRemoveFromStore() throws Exception {
             return null;
         }
 
-        DeleteRequestImpl request = new DeleteRequestImpl(ids.toArray(new String[numberOfMetacardsToRemove]));
+        DeleteRequestImpl request =
+                new DeleteRequestImpl(ids.toArray(new String[numberOfMetacardsToRemove]));
```

Comment:
Just for everyones knowledge who missed the session...Technically just `toArray()` and the `Object[]` versions are the fastest, but you would have to cast the result. I doubt its worth obfuscating this code though since this is probably the least of our performance concerns. 😄